### ISSUE TITLE
Script make.sh does not convert SVG images when launched from its root directory

### ISSUE THREAD

**Issue by deluxe** [fabiensanglard/gebbwolf3#7]
Script does not convert SVG images when launched from its root directory and outputs an error message instead.

- **Steps to reproduce**
Launch the compilation with:
```bash
cd gebbwolf3/ && ./make.sh 
```

- **Result**
The script outputs an error message:
```
find: screenshots_svg: No such file or directory
```

- **Expected result**
The script should walk through `src/screenshots_svg` and compile SVG images in PNG (if newer)

**Comment by fabiensanglard** [comment]
I am unable to reproduce this issue on my Mac. 

```cd gebbwolf3/ && ./make.sh```

Is the way I have always built and as far as remember it works. Can you elaborate?

**Comment by deluxe** [comment]
Even if your script finishes with a PDF, don't you have an error `find: screenshots_svg: No such file or directory` during the execution? As far as I can tell (without retesting), this part of your build _cannot_ work since you don't have `screenshots_svg/` under `gebbwolf3/`.

Now your memory doesn't lie to you: your build _did_ work the way you remember before you merged this commit: https://github.com/fabiensanglard/gebbwolf3/commit/89f6688ad6bf135f693b7b8cb788faf90ef9c767.
In that commit, suppression of `cd src` line 5 (https://github.com/fabiensanglard/gebbwolf3/commit/89f6688ad6bf135f693b7b8cb788faf90ef9c767#diff-c460e5ad08a6c30a4643bb2b781c97cbe49aa128e31909d409936e17d573bf5dL5) means PNG generation doesn't work anymore.
You may not be aware of that if you have previously generated PNGs in workspace. Since you committed already generated PNGs in your Git repository, it's a possibility =)

Please remove generated PNGs and test again (you may also believe me without any further test).

**Comment by fabiensanglard** [comment]
I just tested again and the build works as expected and accordingly to the README.md.

(89f6688#diff-c460e5ad08a6c30a4643bb2b781c97cbe49aa128e31909d409936e17d573bf5dL5) does not "remove" the "cd src", it moves it from line 5 to 65.

**Comment by deluxe** [comment]
I just re-tested at your last commit (current HEAD) with `cd gebbwolf3/ && ./make.sh | head` (with `| head` only to stop the output from flooding the screen).

The resulting output is:
```
find: screenshots_svg: No such file or directory
This is pdfTeX, Version 3.14159265-2.6-1.40.21 (TeX Live 2020) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./book.tex
LaTeX2e <2020-02-02> patch level 5
L3 programming layer <2020-03-06>
(/usr/local/texlive/2020/texmf-dist/tex/latex/base/book.cls
Document Class: book 2019/12/20 v1.4l Standard LaTeX document class
(/usr/local/texlive/2020/texmf-dist/tex/latex/base/bk10.clo)) (./mystyle.sty
(/usr/local/texlive/2020/texmf-dist/tex/latex/subfiles/subfiles.sty
cp: output/book.pdf: No such file or directory
```
Note on the first line `find: screenshots_svg: No such file or directory`.
Can you reproduce it?

You are correct that `cd src` was not really removed but only moved from line 5 to 65 in commit https://github.com/fabiensanglard/gebbwolf3/commit/89f6688ad6bf135f693b7b8cb788faf90ef9c767. But your command to convert SVG from `screenshots_svg` directory was line 58 in that commit, before `cd src` new position.

**Comment by fabiensanglard** [comment]
Fixed in 143e5addead733628df0411f17b01f2804490787
